### PR TITLE
Allow users to --enable-v{1,2} with repo update.

### PR DESCRIPTION
**Pull request by rbarlow**
Our Docker CLI extensions had an oversight where users could not
update the --enable-v{1,2} flags on their repositories. This
commit fixes that oversight.

https://pulp.plan.io/issues/1710

fixes #1710

Instead of unit tests I added a pulp-smash test to ensure that this works correctly (and stays working):

https://github.com/PulpQE/pulp-smash/pull/129

### CODE REVIEW

**Comment by rbarlow** [comment]
The Jenkins failure seems to be a qpid issue and I believe it is not related to this change.

File: extensions_admin/pulp_docker/extensions/admin/cudl.py [hunk 1]
```diff
@@ -57,7 +57,39 @@
...
+
+        :param user_input:  dictionary of data passed in by okaara
+        :type  user_input:  dict
+
+        :return:    importer config
+        :rtype:     dict
+        """
+        config = self.parse_user_input(user_input)
+
+        print user_input
```
Comment (asmacdo) [inline extensions_admin/pulp_docker/extensions/admin/cudl.py]: ^ rm

**Comment by asmacdo** [comment]
Remove that dev artifact and LGTM

File: extensions_admin/pulp_docker/extensions/admin/cudl.py (hunk 1 above)
Comment (rbarlow) [inline extensions_admin/pulp_docker/extensions/admin/cudl.py]: #ugh i'm usually good at not doing that. ☹

**Pull request merged**